packages/common/src/constraints/CheckCandidates.ts

```diff
@@ -124,6 +125,11 @@ export const checkCandidate = async (
       logger.info(`${candidate.name} provider not valid`, constraintsLabel);
     }
 
+    const geoAreaSanctionsValid =
+      constraints.config?.constraints?.skipSanctionedCountries == true
```

Comment:
why you defined skipSanctionedCountries and skipSanctionedRegions but you are checking only Countries ? couldn't you just define a skipSanctionedArea flag ?